src/Microsoft.R.Host.vcxproj

```diff
@@ -130,7 +125,6 @@
       <SDLCheck>false</SDLCheck>
       <AdditionalIncludeDirectories>..\lib\picojson;..\lib\websocketpp</AdditionalIncludeDirectories>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
-      <DisableSpecificWarnings>4200</DisableSpecificWarnings>
```

Comment:
This is suppressing a warning for `class message` in `message.h` - probably still necessary.